implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -53,7 +55,19 @@
      * @return a new instance of JWTAuthContextInfoProvider
      */
     public static JWTAuthContextInfoProvider createWithKey(String publicKey, String issuer) {
-        return create(publicKey, NONE, issuer);
+        return create(publicKey, NONE, NONE, issuer);
+    }
+
+    /**
+     * Create JWTAuthContextInfoProvider with the public key and issuer
+     * 
+     * @param secret - Base64 HMAC shared secret
+     *
+     * @param issuer the issuer
+     * @return a new instance of JWTAuthContextInfoProvider
+     */
+    public static JWTAuthContextInfoProvider createWithHmac(String secret, String issuer) {
```

Comment:
IMHO we only need one extra overload, `createWtithSecretKey` which would take `secretKeyLocation` and `issuer` parameters. We might add a support for the inline private keys later...